adb_graphics/default_specs.yml

```diff
@@ -1326,6 +1344,12 @@ u:
   max:
     ncl_name: MAXUW_P8_L103_{grid}_max1h
     transform: conversions.ms_to_kt
+  lvl5:
```

Comment:
do these winds need to be converted to AGL?  I know we have transforms to AGL for other fields like Aviation Flight Rules, ceiling, etc.  There seem to be difference in the NCL v python winds, especially over the mountains.

Reply:
It turns out that these levels for the python plots are one level higher than they should be.  Please see my reply to Christina for the details.